features/netsocket/emac-drivers/TARGET_STM_EMAC/TARGET_STM32F4/TARGET_UBLOX_EVK_ODIN_W2/wifi_emac/wifi_emac.h

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2017 ARM Limited. All rights reserved.
+ */
+
+#ifndef WIFI_EMAC_H_
+#define WIFI_EMAC_H_
+
+#include "EMAC.h"
+#include "cb_wlan_target_data.h"
+#include "cb_wlan.h"
+
+class WIFI_EMAC : public EMAC {
```

Comment:
I think this should be named a bit more specifically than `WIFI_EMAC`. Could easily collide. `OdinWiFiEMAC`?

Reply:
ok. yes this seems fine.

@andreaslarssonublox  what do you think?

Reply:
Agree